examples/hyperparam/train.py

```diff
@@ -112,11 +112,7 @@ def on_epoch_end(self, epoch, logs=None):
 @click.argument("training_data")
 def run(training_data, epochs, batch_size, learning_rate, momentum, seed):
     warnings.filterwarnings("ignore")
-    wine_url = (
-        "http://archive.ics.uci.edu/ml/machine-learning-databases/"
-        "wine-quality/winequality-white.csv"
-    )
-    data = pd.read_csv(training_data) if training_data != '' else pd.read_csv(wine_url, sep=';')
+    data = pd.read_csv(training_data, sep=';')
```

Comment:
:O This is awesome! Had no idea the `pandas.read_csv` supported this! Thanks @harupy